personal-brand/personal-branding.md

```diff
@@ -1,73 +1,178 @@
 # Developing your brand as a software developer
 
-### Soft Skills for Week 2
-
+## Soft Skills for Week 2
 - What are your personal values, and how does your personal brand reflect that?
 - 3 parts of the soft skills answer:
   - Intro - a super short summary of what you will be talking about - give me the punch line right away
   - Story - only tell details that show what you did!
-  - Take away “What I learned from this”, “Going forward I will always”, “The lesson I learned from this situation”...
-
-### Prerequisites
+  - Take away "What I learned from this", "Going forward I will always", "The lesson I learned from this situation"...
 
+## Prerequisites
 - [GitHub account](https://github.com/)
-- [Twitter (or X) account](https://twitter.com)
-- [LinkedIn account](https://linkedin.com)
+- Professional social media presence on at least one of these platforms:
+  - [LinkedIn account](https://www.linkedin.com/)
+  - [Mastodon](https://joinmastodon.org/)
+  - [BlueSky](https://bsky.app/)
+  - [Threads](https://www.threads.net/)
 
-### Motivation
+## Motivation
 
 - Your personal brand is your way to distinguish yourself from others and make yourself memorable.
 - Having a distinct and relevant personal brand can help employers identify you as a good fit for their open roles, or can help you build credibility with colleagues and managers as you further your career.
 - It is worth the investment to think outside the box and develop a brand that will help employers get to know you.
+- In today's digital landscape, thoughtfully managing your online presence is crucial for career development and networking.
```

Comment:
We might mention something here about helping employers establish authenticity if you end up needing to apply for roles for which you don't have referrals, unless that seems too hostile to mention upfront here to you?